index.html

```diff
@@ -4,9 +4,99 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Hello, YOUR_NAME</title>
+  <title>Hello, Kendahl</title>
+  <style>
+
+html {
+  background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+	background-size: 400% 400%;
+	animation: gradient 10s ease infinite;
+}
+
+@keyframes gradient {
+	0% {
+		background-position: 0% 50%;
+	}
+	50% {
+		background-position: 100% 50%;
+	}
+	100% {
+		background-position: 0% 50%;
+	}
+}
+
+  div.gallery {
+    margin: 5px;
+    border: 1px solid gold;
+    float: left;
+    width: 180px;
+    /* filter: grayscale(80%) */
+  }
+  div.gallery img {
+  width: 100%;
+  height: auto;
+  /* transform: skew(10deg) translate(150px);
+  transform-origin: bottom left; */
+}
+
+div.gallery:hover {
+  border: 3px solid purple;
+}
+
+  </style>
 </head>
 <body>
-  <!-- use this space to add your own content -->
+  <h1><center>Welcome to the Danga Zone!!!</center></h1>
```

Comment:
1. Use `text-align` in CSS to change display-related properties like text alignment.
2. An `<h1>` containing a `<center>` tag is invalid HTML, so make sure to use the validator to double check.